channels.nix

```diff
@@ -1,87 +1,114 @@
 rec {
   channels = {
-      # "Channel name" = {
-      #   job = "project/jobset/jobname"; like https://hydra.nixos.org/job/<value>/latest-finished
-      #   # when adding a new version, mark the oldest as "not current". New releases should always be current.
-      #   # Channels where `current = false` are marked `end-of-life` on https://status.nixos.org/ and alerting is disabled.
-      #   current = true;
-      # };
-      "nixos-unstable" = {
-        job = "nixos/trunk-combined/tested";
-        current = true;
-      };
-      "nixos-unstable-small" = {
-        job = "nixos/unstable-small/tested";
-        current = true;
-      };
-      "nixpkgs-unstable" = {
-        job = "nixpkgs/trunk/unstable";
-        current = true;
-      };
-      
-      "nixos-21.11" = {
-        job = "nixos/release-21.11/tested";
-        current = true;
-      };
-      "nixos-21.11-small" = {
-        job = "nixos/release-21.11-small/tested";
-        current = true;
-      };
-      "nixpkgs-21.11-darwin" = {
-        job = "nixpkgs/nixpkgs-21.11-darwin/darwin-tested";
-        current = true;
-      };
-      "nixos-21.11-aarch64" = {
-        job = "nixos/release-21.11-aarch64/tested";
-        current = true;
-      };
+    # "Channel name" = {
+    #   job = "project/jobset/jobname"; # like https://hydra.nixos.org/job/<value>/latest-finished
+    #   # When adding a new version, determine if it needs to be tagged as a
+    #   # variant -- for example, nixos-21.05, nixos-21.05-small,
+    #   # nixos-21.05-aarch64, and nixpkgs-21.05-darwin would be variants
+    #   # "primary", "small", "aarch64", and "darwin", respectively.
+    #   variant = "primary";
+    #   # When adding a new version, determine the status of the channel -- for
+    #   # example, unreleased NixOS versions would be "beta", released NixOS
+    #   # versions would be "stable", NixOS versions that are still supported
+    #   # but no longer the latest stable would be "deprecated", rolling
+    #   # release channels would be "rolling", channels that are no longer
+    #   # maintained but not yet removed from this list would be marked as
+    #   # "unmaintained".
```

Comment:
```suggestion
    #
    #   # Channel Status:
    #   # '*-unstable' channels are always "rolling"
    #   # Otherwise a release progresses through phases:
    #   #
    #   #  - Directly after branch off                   => "beta"
    #   #  - Once the channel is released                => "stable"
    #   #  - Once the next channel is released           => "deprecated"
    #   #  - 3 months after the next channel is released => "unmaintained"
```